### PR TITLE
SRVCOM-1288 + SRVCOM-1664: Clean up attributes, prereqs

### PR DESCRIPTION
**Pull request by abrennan89**
Jiras: https://issues.redhat.com/browse/SRVCOM-1288 + https://issues.redhat.com/browse/SRVCOM-1664

Applies for OCP 4.6+

Preview links:
- https://deploy-preview-41839--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-ossm-setup.html#serverless-ossm-enabling-serving-metrics_serverless-ossm-setup
- https://deploy-preview-41839--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-ossm-setup.html#serverless-ossm-setup-with-kourier_serverless-ossm-setup
- https://deploy-preview-41839--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-ossm-setup.html#serverlesss-ossm-external-certs_serverless-ossm-setup
- https://deploy-preview-41839--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-ossm-setup.html#serverless-ossm-setup_serverless-ossm-setup

(not much to see, just adding / updating prereqs)

### CODE REVIEW

**Comment by skonto** [comment]
It might be a bigger change (another PR) but can we remove the dots as in here?

image

When I copy paste the snippet I get the dots too which is annoying. It is easier to have ready to use code.

File: serverless/admin_guide/serverless-ha.adoc [hunk 1]
```diff
@@ -29,7 +30,6 @@ These components are configured with two replicas by default.
 For Knative Eventing, the `mt-broker-filter` and `mt-broker-ingress` deployments are not scaled by HA. If multiple deployments are needed, scale these components manually.
 
 You modify the number of replicas that are created per component by changing the configuration of `spec.high-availability.replicas` in the KnativeServing custom resource (CR), the KnativeEventing CR, or the KnativeKafka CR.
-// This field also specifies the minimum number of _activators_ if you are using the horizontal pod autoscaler (HPA). For more information about HPA, see
```
Comment (skonto) [inline serverless/admin_guide/serverless-ha.adoc:32]: Why we needed to document this? Seems a bit odd to reference only activators.

**Comment by abrennan89** [comment]
> It might be a bigger change (another PR) but can we remove the dots as in here?
> 
> image
> 
> When I copy paste the snippet I get the dots too which is annoying. It is easier to have ready to use code.

The dots are part of the OCP docs guidelines that we use now to show that it's a partial YAML file / snippet and not a full example. "For snippets or sections of a file, use an ellipsis (…​) to show that the file continues before and/or after the quoted block." see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

**Comment by jrangelramos** [comment]
QE Approved

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**